packages/wallet-sdk/tests/generate.test.ts

```diff
@@ -70,3 +78,64 @@ describe(generateWallet, () => {
     );
   });
 });
+
+describe('Compatibility verification @scure/bip39 vs bitcoinjs/bip39', () => {
+  test('Verify compatibility @scure/bip39 <=> bitcoinjs/bip39', () => {
+    // Consider an entropy
+    const entropy = '00000000000000000000000000000000';
+    // Consider same entropy in array format
+    const entropyUint8Array = new Uint8Array(entropy.split('').map(Number));
+
+    // Use vectors to verify result with bitcoinjs/bip39 instead of importing bitcoinjs/bip39
```

Comment:
Is there a reason this test mocks `bitcoinjs/bip39` rather than using the actual one as a devDependency?

Reply:
> Is there a reason this test mocks `bitcoinjs/bip39` rather than using the actual one as a devDependency?

https://github.com/hirosystems/stacks.js/pull/1218#discussion_r841955064
<img width="841" alt="Screenshot 2022-04-28 at 4 37 17 PM" src="https://user-images.githubusercontent.com/23546098/165743882-b5c61909-f95e-4178-8dac-7f8547f37f37.png">